pages/404.tsx

```diff
@@ -0,0 +1,9 @@
+import PageNotFound from '../src/PageNotFound';
+
+function FourOhFour() {
+    return (
+      <PageNotFound />
+    )
+  }
+  
+  export default FourOhFour
```

Comment:
EOL..

Seems like we are not running format at any point.
We probably should add the formatting step before pre-commit and within the vscode setting - formatOnSave

Reply:
A step in the CI to enforce the styles would be awesome!  I'll look into that.